src/AutoRest.CSharp/Mgmt/Output/ResourceContainer.cs

```diff
@@ -31,11 +34,34 @@ public ResourceContainer(OperationGroup operationGroup, BuildContext<MgmtOutputL
         protected override string CreateDescription(OperationGroup operationGroup, string clientPrefix)
         {
             StringBuilder summary = new StringBuilder();
```

Comment:
it appears this variable isn't used anywhere

Reply:
You're right, just checked and there are no references for that variable.